resources/validations/src/ssp.sch

```diff
@@ -1375,10 +1372,10 @@ system-implementation user assembly.</sch:assert>
                         id="user-has-user-type"
                         role="error"
                         test="oscal:prop[@name = 'type']">Every user has a user type property.</sch:assert>
-            <sch:assert diagnostics="user-has-privilege-type-diagnostic"
-                        id="user-has-privilege-type"
+            <sch:assert diagnostics="user-has-privilege-level-diagnostic"
+                        id="user-has-privilege-level"
                         role="error"
-                        test="oscal:prop[@name = 'privilege-type']">Every user has a privilege type property.</sch:assert>
+                        test="oscal:prop[@name = 'privilege-level']">Every user has a privilege type property.</sch:assert>
```

Comment:
NB: Thank for fixing this. Did this not break in tests because of how we write tests? 🤔

Reply:
The XSpec context agreed with the incorrect prop name.

Reply:
OK it seems I misunderstood the tests then.